src/promptflow-evals/CHANGELOG.md

```diff
@@ -5,7 +5,19 @@
 ### Features Added
 - Introduced `JailbreakAdversarialSimulator` for customers who need to do run jailbreak and non jailbreak adversarial simulations at the same time. More info in the README.md in `/promptflow/evals/synthetic/README.md#jailbreak-simulator`
 - Exposed batch evaluation run timeout via `PF_BATCH_TIMEOUT_SEC` environment variable. This variable can be used to set the timeout for the batch evaluation for each evaluator and target separately only, not the entire API call.
+- Support for Tier 1 languages in the simulator.
 
+Here's how you can use it:
+```python

```

Comment:
Do we have a culture of adding examples inside the CHANGELOG notes? 

If not, could we add this to the SupportedLanguages docstring instead? That way it shows up along with the reference documentation.